### PR TITLE
completed login page feature

### PR DESCRIPTION
**Pull request by Tanvir04khan**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+
+import Login from "./components/Login/Login";
+
+import "./App.css";
+
+function App() {
+  return (
+    <div>
```
Comment (Rehmatulla-Khan) [inline src/App.jsx:9]: Can we use fragment here ?

**Review by Rehmatulla-Khan — CHANGES_REQUESTED** [review]

File: src/App.jsx (hunk 1 above)
Comment (Tanvir04khan) [reply src/App.jsx:9]: maybe, he would do something with this <div>.

File: src/components/Login/Login.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        </a>
+        <div>
+          <Join>Join now</Join>
+          <SignIn>Sign in</SignIn>
+        </div>
+      </Nav>
+      <Section>
+        <Hero>
+          <h1>Welcome to your professional community</h1>
+          <img src={heroSvg} alt="" />
```
Comment (faran4engg) [inline src/components/Login/Login.jsx]: alt ?

File: src/components/Login/Login.styled.jsx [hunk 3]
```diff
@@ -0,0 +1,180 @@
+import styled from "styled-components";
+
+export const Container = styled.div`
+  padding: 0px;
```
Comment (faran4engg) [inline src/components/Login/Login.styled.jsx]: it can be 0 without unit

File: src/index.css [hunk 4]
```diff
@@ -1 +1,23 @@
...
+  margin: 0;
+  padding: 0;
+
+  outline: none;
+
+  font-size: 100%;
+  vertical-align: baseline;
+
+  background: transparent;
+}
```
Comment (faran4engg) [inline src/index.css:15]: move your app.css code here and remove that file completely

File: src/index.css [hunk 5]
```diff
@@ -1 +1,23 @@
...
 
+:root,
+body,
+html {
+  background: #f5f5f5;
+
+  box-sizing: border-box;
+}
+* {
```
Comment (faran4engg) [inline src/index.css:3]: this should come before body

**Review by faran4engg — APPROVED** [review]
fix then merge

**Pull request merged**